src/opm/parser/eclipse/EclipseState/Tables/TableManager.cpp

```diff
@@ -1441,6 +1455,18 @@ std::optional<JFunc> make_jfunc(const Deck& deck) {
         assert(regionIdx == numTables);
     }
 
+    template <class TableType>
+    void TableManager::initSaltsolTables(const Deck& deck,  std::vector<TableType>& saltsoltables) {
+        size_t numTables = m_tabdims.getNumPVTTables();
+        saltsoltables.resize(numTables);
+
+        const auto& keyword = deck.getKeyword("SALTSOL");
```

Comment:
This will not compile any longer:
```C++
#include <opm/parser/eclipse/Parser/ParserKeywords/S.hpp>
...
const auto& keyword = deck.get<ParserKeywords::S::SALTSOL>();
...
```